.travis.yml

```diff
@@ -1,4 +1,7 @@
+os: linux
+dist: xenial
```

Comment:
Xenial is getting a bit old now isn't it? I think it'll be two LTS releases back as of Thursday. Do you think it's worth it to try bumping this to bionic?

Reply:
Yes, good point. I had initially set it to bionic, but was messing around with things and forgot to change it back...